client/src/App.scss

```
@@ -121,6 +121,35 @@ Landing overview
   padding-right: 1rem;
 }
 
+.loginBtns {
+  display: flex;
+  width: 30%;
+  justify-content: space-around;
+  margin-bottom: 2 * $unit;
+}
```

hmm this is a strange way of doing this (using space-around, then shrinking the container really small to get it to align).

I would just `display: flex;` and then add a margin to the second button, and then reuse these same classes on the users buttons

```suggestion
.btnContainer {
  display: flex;
  margin-bottom: 2 * $unit;
}

.btnWithMargin {
  margin-left: 2 * $unit;
}
```